apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -249,6 +261,9 @@ const Ticket = new GQLListSchema('Ticket', {
     },
 })
 
+const fieldsToTrackInTicketForChanges = trackableFieldsFrom(Ticket.schema, { except: OMIT_TICKET_CHANGE_TRACKABLE_FIELDS })
+
 module.exports = {
     Ticket,
+    fieldsToTrackInTicketForChanges,
```

Comment:
Why you want to export it? Usually we export only models

Reply:
Fixed in `d393fce9`.